src/infra/db/mssql/util/knex/extensions/turbo-interceptor.ts

```diff
@@ -0,0 +1,165 @@
+import k, { type Knex } from 'knex';
+import NodeCache from 'node-cache';
+
+import { logger } from '@/util';
+import { makeCacheServer } from '@/infra/cache';
+
+type Services = 'memjs' | 'node-cache';
+type GenericObject = Record<string, unknown>;
+
+const cache = new NodeCache({ stdTTL: 100, checkperiod: 120 });
+const memCache = makeCacheServer();
+
+const MAX_MEMORY_USAGE = 100 * 1024 * 1024; // 100 MB
```

Comment:
This won't work. When running in production, this project consumes an average of 125MB under normal usage. For it to function correctly, your limit has to be at least 300MB.

Reply:
thanks for the context, I'll update the value